editor/scene_tree_dock.cpp

```diff
@@ -1749,17 +1763,26 @@ void SceneTreeDock::_toggle_placeholder_from_selection() {
 	}
 }
 
-void SceneTreeDock::_toggle_editable_children(Node *p_node) {
-	if (p_node) {
-		bool editable = !EditorNode::get_singleton()->get_edited_scene()->is_editable_instance(p_node);
-		EditorNode::get_singleton()->get_edited_scene()->set_editable_instance(p_node, editable);
-		if (editable) {
-			p_node->set_scene_instance_load_placeholder(false);
-		}
+void SceneTreeDock::_toggle_editable_children(Node *p_node, Node *p_owner, bool p_edit_all) {
+	if (!p_node) {
+		return;
+	}
+	bool editable = !EditorNode::get_singleton()->get_edited_scene()->is_editable_instance(p_node);
+	bool root_editable = EditorNode::get_singleton()->get_edited_scene()->is_editable_instance(p_owner);
+	editable |= !root_editable;
```

Comment:
While it works, I'm not sure it's a good practice per se to use a bitwise operator for booleans.

Reply:
Expanded to editable = editable || !root_editable;